static/app/components/replays/replayView.tsx

```diff
@@ -22,7 +25,7 @@ function ReplayView({toggleFullscreen}: Props) {
       <Panel>
         <ReplayPlayer />
       </Panel>
-      <ReplayController toggleFullscreen={toggleFullscreen} />
+      {!hasNewTimeline ? <ReplayController toggleFullscreen={toggleFullscreen} /> : null}
```

Comment:
Nit!!!

Usually it's easier to read conditions that are not negated. I'd flip the order of the ternary to keep the logic the same:
```suggestion
      {hasNewTimeline ? null : <ReplayController toggleFullscreen={toggleFullscreen} />}
```

So it reads more like "If has new timeline, then show nothing, else show the controller"


the change in the other file has the same condition, but different result. In english it's "if has new timeline, then show the controller, else show nothing"